bazel/utils/diff_test.bzl

```diff
@@ -61,32 +109,38 @@ def _diff_test_impl(ctx):
 diff_test = rule(
     doc = """
       A test that compares the contents of two files to ensure they are
-      identical.  Typically, this would be used to compare the output of a
```

Comment:
the revised docstring is more generic and suitable for external use.  I'll port this back into internal if you don't change it.

Reply:
makes sense, done.